production/loki-mixin/config.libsonnet

```diff
@@ -67,12 +70,12 @@
     // Tunes histogram recording rules to aggregate over this interval.
     // Set to at least twice the scrape interval; otherwise, recording rules will output no data.
     // Set to four times the scrape interval to account for edge cases: https://www.robustperception.io/what-range-should-i-use-with-rate/
-    recording_rules_range_interval: '1m',
+    recording_rules_range_interval: '3m',
 
     // SSD related configuration for dashboards.
     ssd: {
       // Support Loki SSD mode on dashboards.
-      enabled: false,
+      enabled: true,
```

Comment:
i think let's also leave this as false, in general these days we are encouraging folks to use the distributed mode now that the helm chart supports it.